governance/program/src/processor/process_create_account_governance.rs

```diff
@@ -29,10 +29,13 @@ pub fn process_create_account_governance(
     let realm_info = next_account_info(account_info_iter)?; // 0
     let account_governance_info = next_account_info(account_info_iter)?; // 1
     let governed_account_info = next_account_info(account_info_iter)?; // 2
-    let payer_info = next_account_info(account_info_iter)?; // 3
-    let system_info = next_account_info(account_info_iter)?; // 4
 
-    let rent_sysvar_info = next_account_info(account_info_iter)?; // 5
+    let _token_owner_record_info = next_account_info(account_info_iter)?; // 3
```

Comment:
What are all of these for? Is it worth adding any checks even at this point or is it ok for junk to be provided?